docs/workloads/azure/wordpress-azure.md

```diff
@@ -12,7 +12,7 @@ In this environment, the following components will be dynamically provisioned an
 Before starting this guide, you should have already [configured your Azure account](../../cloud-providers/azure/azure-provider.md) for usage by Crossplane.
 
 - You should have a `crossplane-azure-provider-key.json` file on your local filesystem, preferably at the root of where you cloned the [Crossplane repo](https://github.com/crossplaneio/crossplane).
-- You should have a azure resource group with name `group-westus-1`. If not, change the value of `resourceGroupName` to an existing resource group in `cluster/examples/workloads/wordpress-azure/provider.yaml`
+- You should have a azure resource group with name `group-westus-1`. If not, change the value of `resourceGroupName` to an existing resource group in `cluster/examples/workloads/wordpress/azure/provider.yaml`
```

Comment:
does this `cluster/examples/workloads/wordpress/azure/` path exist? I don't see it in https://github.com/soorena776/crossplane/tree/aws_wordpress_workload/cluster/examples

It's mentioned a few times in this doc.

Reply:
it does not currently. my assumption is that it's a draft, and I updated accordingly

Reply:
At one point, these docs were working, e.g. in the v0.2 release these paths used to exist:
https://github.com/crossplaneio/crossplane/tree/release-0.2/cluster/examples/workloads/wordpress-azure

We have not done a good job at keeping the user guides up to date and we have regressed them multiple times, we need to improve on that.  We should fix this up properly in #650

Reply:
I had that change here https://github.com/crossplaneio/crossplane/pull/574/files#diff-13c4c083ad9c3b929a0d39dae2d49326R15 (although it has changed again since my draft PR).

I couldn't get Azure creds working to finish the PR before moving off to Stacks work.  There may be other pieces of that PR that can be picked up here or in #650.